src/app/components/addOffer/AddOffer.css

```
@@ -0,0 +1,548 @@
+  color: #37474f;
+  display: flex;
+}
+
+.partnersText {
+  color: #757575;
+  font-size: 23px;
+  font-weight: 500;
+  letter-spacing: 1.16px;
+  line-height: 31px;
+  display: flex;
+  flex-direction: column;
+  padding-top: 67px;
+}
+
+.partnersText h1 {
+  font-size: 46px;
+  letter-spacing: 1.08px;
+  line-height: 57px;
+  font-weight: 600;
+  text-align: left;
+  background: none;
+  min-height: 80px;
+  box-sizing: border-box;
+  font-family: 'Open Sans', 'Helvetica', 'Arial', sans-serif;
+  color: #37474f;
+}
+
+.partnersText p {
+  font-weight: bold;
+  margin-top: 10px;
+}
+
+.partnersImg {
+  height: 100%;
+  display: flex;
+  justify-content: flex-start;
+}
+
+.partnersRow {
+  display: flex;
+  justify-content: center;
+}
+
+.cardSection {
+  margin-right: 40px;
+  background-color: white;
+  width: 500px;
+  box-shadow: 0 15px 30px 0 rgba(214, 231, 249, 0.55);
+  border-radius: 10px;
+  font-size: 24px;
+  letter-spacing: 0.38px;
+  line-height: 36px;
+  padding: 32px 20px 32px 56px;
+}
+
+.cardText {
+  color: #37474f;
+}
+
+.card-number {
+  color: #ff66b3;
+  font-weight: 600;
+}
+
+.moreAboutUs {
+  width: 100%;
+  background-color: white;
+  overflow: hidden;
+  border-bottom: 1px solid grey;
+}
+
+.moreMain {
+  max-width: 1250px;
+  border-radius: 4px;
+  padding-top: 67px;
+  background: none;
+  margin: 0 auto;
+  display: flex;
+}
+
+.textCol h1 {
+  font-size: 46px;
+  font-weight: 600;
+  letter-spacing: 1.08px;
+  line-height: 57px;
+  text-align: left;
+  color: #37474f;
+}
+
+.items {
+  width: 100%;
+}
+
+.imgCol {
+  width: 50%;
+  margin-top: 100px;
+}
+
+.item {
+  display: flex;
+  align-items: center;
+  margin: 36px 0;
+  width: 530px;
+}
+
+.moreLogo {
+  margin-right: 20px;
+}
+
+.moreText {
+  font-size: 20px;
+  letter-spacing: 0.47px;
+  line-height: 24px;
+  color: #37474f;
+}
+
+.moreText b {
+  color: #37474f;
+}
+
+.buttonCenter {
+  margin-top: 40px;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.foot {
+  width: 300px;
+}
+
+.moreFooter {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin-bottom: 40px;
+  border-top: 1px solid lightgrey;
+}
+
+.madeBy {
+  display: flex;
+  justify-content: center;
+  width: 400px;
+  font-weight: 600;
+  font-size: 24px;
+  color: #37474f;
+  margin-bottom: 40px;
+}
+
+.madeBy p {
+  color: #ff66b3;
+  margin: 0;
+  margin-right: 10px;
+}
+
+.moreLinks {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.moreLinks a {
+  text-decoration: none;
+  font-size: 20px;
+  font-weight: 400;
+  margin-left: 30px;
+  letter-spacing: 0.47px;
+  line-height: 24px;
+  color: #757575;
+}
+
+.moreLinks i {
+  margin-left: 5px;
+  font-size: 16px;
+}
+
+.range-sliderDescript {
+  display: flex;
+  align-items: flex-start;
+}
+
+.carousel-dots li.carousel-dots-active button:before {
+  color: #ff4081 !important;
+}
+
+.carousel-dots li.carousel-dots-active button:hover {
+  color: #ff4081 !important;
+}
```

```suggestion
.carousel-dots li.carousel-dots-active button:before,
.carousel-dots li.carousel-dots-active button:hover {
  color: #ff4081 !important;
}
```
ponadto warto używać zmiennych CSS:

Deklaracja: `--nazwa-zmiennej: wartość;`,
użycie: `var(--nazwa-zmiennej);`